scripts/lxc-to-lxd

```diff
@@ -152,18 +246,6 @@ def convert_container(lxd_socket, container_name, args):
         print("Container name doesn't match lxc.utsname, skipping...")
         return False
 
-    # Detect privileged containers
-    print("Validating container mode")
-    if config_get(lxc_config, "lxc.id_map"):
-        print("Unprivileged containers aren't supported, skipping...")
-        return False
```

Comment:
I think we should keep that check and run it before the generic one as the custom error messages makes it worth it (people are unlikely to know what lxc.id_map is).